src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -116,39 +116,6 @@ public void AnalyzeTemplate_ValidInputValues_AnalyzesUsingAutoDetectedParameters
             StringAssert.Contains(outputWriter.ToString(), "Parameters File: " + Path.Combine(Directory.GetCurrentDirectory(), "Tests", "ToTestSeparateParametersFile", "TemplateWithSeparateParametersFile.parameters.json"));
         }
 
-        [TestMethod]
-        public void AnalyzeDirectory_ValidInputValues_AnalyzesExpectedNumberOfFiles()
-        {
-            var args = new string[] { "analyze-directory", Path.Combine(Directory.GetCurrentDirectory(), "Tests") };
-
-            using StringWriter outputWriter = new();
-            Console.SetOut(outputWriter);
-
-            var result = _commandLineParser.InvokeCommandLineAPIAsync(args);
-
-            Assert.AreEqual((int)ExitCode.ErrorAndViolation, result.Result);
-            StringAssert.Contains(outputWriter.ToString(), "Analyzed 14 files");
-        }
-
-        [TestMethod]
-        public void AnalyzeDirectory_ValidInputValues_AnalyzesExpectedNumberOfFilesWithAutoDetectedParameters()
-        {
-            var args = new string[] { "analyze-directory", Path.Combine(Directory.GetCurrentDirectory(), "Tests", "ToTestSeparateParametersFile") };
-
-            using StringWriter outputWriter = new();
-            Console.SetOut(outputWriter);
-
-            var result = _commandLineParser.InvokeCommandLineAPIAsync(args);
-
-            Assert.AreEqual((int)ExitCode.Success, result.Result);
-
-            StringAssert.Contains(outputWriter.ToString(), "Analyzed 4 files");
-            StringAssert.Contains(outputWriter.ToString(), "Parameters File: " + Path.Combine(Directory.GetCurrentDirectory(), "Tests", "ToTestSeparateParametersFile", "TemplateWithSeparateParametersFile.parameters.json"));
```

Comment:
I also don't see another test to validate that we are printing results for the pairs found